I had a few issues with this release:

1) x64-linux,x64-windows `[core,npu]` configuration failure:
```log
CMake Error at thirdparty/flatbuffers/CMakeLists.txt:18 (add_subdirectory):
  The source directory
    /mnt/vcpkg-ci/b/openvino/src/2026.0.0-b918f7b76f.clean/thirdparty/flatbuffers/flatbuffers
  does not contain a CMakeLists.txt file.
```
`npu` now requires `flatbuffers`, need to update portfile and fix openvino build scripts to allow it

2) x64-linux `all;[core,gpu]` post-build validation failure:
```log
/vcpkg/ports/openvino/portfile.cmake: warning: There should be no absolute paths, such as the following, in an installed package. To suppress this message, add set(VCPKG_POLICY_SKIP_ABSOLUTE_PATHS_CHECK enabled)
note: /mnt/vcpkg-ci/p
note: /mnt/vcpkg-ci/installed
note: /mnt/vcpkg-ci/b
note: /vcpkg/downloads
/mnt/vcpkg-ci/p/openvino_x64-linux_4/share/openvino/OpenVINOTargets.cmake: note: absolute paths found here
```
Turns out `onednn_npu_includes.patch` is still needed.

3) x64-windows `[core,gpu]` dbg install failure:
```log
FAILED: [code=1248] D:/b/openvino/src/2026.0.0-b918f7b76f.clean/bin/intel64/Debug/openvino_intel_gpu_graphd.lib 
C:\Windows\system32\cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:D:\b\openvino\src\2026.0.0-b918f7b76f.clean\bin\intel64\Debug\openvino_intel_gpu_graphd.lib @CMakeFiles\openvino_intel_gpu_graph.rsp && cd ."
D:\b\openvino\src\2026.0.0-b918f7b76f.clean\bin\intel64\Debug\openvino_intel_gpu_graphd.lib : fatal error LNK1248: image size (100B121A5) exceeds maximum allowable size (FFFFFFFF)
ninja: build stopped: subcommand failed.
```
To be investigated

4) x64-windows configuration failure:
```log
-- Unzipping prebuilt Plugin compiler libraries to D:/b/openvino/src/2026.0.0-b918f7b76f.clean/src/plugins/intel_npu/temp/plugin_compiler_lib/win/npu_compiler_vcl_windows_2022-7_5_0-a1ae54e9
CMake Error: Problem with archive_write_header(): Can't unlink already-existing object
CMake Error: Current file: cid/lib/npu_driver_compiler.dll
CMake Error at src/plugins/intel_npu/cmake/download_compiler_libs.cmake:39 (file):
  file failed to extract:
  D:/b/openvino/src/2026.0.0-b918f7b76f.clean/src/plugins/intel_npu/temp/plugin_compiler_lib/win/npu_compiler_vcl_windows_2022-7_5_0-a1ae54e9.zip
```
Might be a problem with parallel configuration, need to download/extract to configuration directory? Attempted to fix it by replacing the sources directory with binary directory.

5) arm64-windows install failure:
```log
D:\b\openvino\src\2026.0.0-b918f7b76f.clean\src\inference\src\system_conf.cpp(472): error C3861: 'GetCurrentProcessorNumber': identifier not found
D:\b\openvino\src\2026.0.0-b918f7b76f.clean\src\inference\src\system_conf.cpp(485): error C3861: 'GetCurrentProcessorNumber': identifier not found
```
Attempted to fix it by adding `<windows.h>` header.

--- a/ports/openvino/npu_deps.patch
+++ b/ports/openvino/npu_deps.patch
@@ -1,0 +1,44 @@
+diff --git a/cmake/features.cmake b/cmake/features.cmake
+index 098f5b752b..bd55827203 100644
+--- a/cmake/features.cmake
++++ b/cmake/features.cmake
+@@ -190,7 +190,7 @@ ov_dependent_option (ENABLE_SYSTEM_TBB  "Enables use of system TBB" ${ENABLE_SYS
+ ov_option (ENABLE_SYSTEM_PUGIXML "Enables use of system PugiXML" OFF)
+ # the option is on by default, because we use only flatc compiler and don't use any libraries
+ ov_dependent_option(ENABLE_SYSTEM_FLATBUFFERS "Enables use of system flatbuffers" ${ENABLE_SYSTEM_FLATBUFFERS_DEFAULT}
+-    "ENABLE_OV_TF_LITE_FRONTEND" OFF)
++    "ENABLE_OV_TF_LITE_FRONTEND OR ENABLE_INTEL_NPU" OFF)
+ ov_dependent_option (ENABLE_SYSTEM_OPENCL "Enables use of system OpenCL" ${ENABLE_SYSTEM_LIBS_DEFAULT}
+     "ENABLE_INTEL_GPU" OFF)
+ # the option is turned off by default, because we compile our own static version of protobuf
+diff --git a/src/plugins/intel_npu/cmake/download_compiler_libs.cmake b/src/plugins/intel_npu/cmake/download_compiler_libs.cmake
+index 89009aa84e..69824f6986 100644
+--- a/src/plugins/intel_npu/cmake/download_compiler_libs.cmake
++++ b/src/plugins/intel_npu/cmake/download_compiler_libs.cmake
+@@ -104,7 +104,7 @@ if(ENABLE_INTEL_NPU_COMPILER)
+         set(NPU_COMPILER_LIB_DESTINATION "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${CMAKE_BUILD_TYPE}")
+     endif()
+     if(WIN32)
+-        set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/temp/plugin_compiler_lib/win")
++        set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_BINARY_DIR}/temp/plugin_compiler_lib/win")
+         set(PLUGIN_COMPILER_LIBS_URL "https://storage.openvinotoolkit.org/dependencies/thirdparty/windows/npu_compiler_vcl_windows_2022-7_5_0-a1ae54e9.zip")
+         set(PLUGIN_COMPILER_LIBS_ZIP "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_windows_2022-${PLUGIN_COMPILER_VERSION}-a1ae54e9.zip")
+         set(PLUGIN_COMPILER_LIBS_DIR_UNZIPPED "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_windows_2022-${PLUGIN_COMPILER_VERSION}-a1ae54e9")
+@@ -131,7 +131,7 @@ if(ENABLE_INTEL_NPU_COMPILER)
+             if(OS_NAME STREQUAL "Ubuntu")
+                 if(OS_VERSION STREQUAL "22.04")
+                     # Ubuntu 22.04-specific settings or actions
+-                    set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/temp/compiler_libs/ubuntu22.04")
++                    set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_BINARY_DIR}/temp/compiler_libs/ubuntu22.04")
+                     set(PLUGIN_COMPILER_LIBS_URL "https://storage.openvinotoolkit.org/dependencies/thirdparty/linux/npu_compiler_vcl_ubuntu_22_04-7_5_0-a1ae54e9.tar.gz")
+                     set(PLUGIN_COMPILER_LIBS_TAR "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_ubuntu_22_04-${PLUGIN_COMPILER_VERSION}-a1ae54e9.tar.gz")
+                     set(PLUGIN_COMPILER_LIBS_DIR_EXTRACTED "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_ubuntu_22_04-${PLUGIN_COMPILER_VERSION}-a1ae54e9")
+@@ -149,7 +149,7 @@ if(ENABLE_INTEL_NPU_COMPILER)
+                     message(STATUS "Copying prebuilt Plugin compiler libraries libopenvino_intel_npu_compiler.so to ${NPU_COMPILER_LIB_DESTINATION} for Ubuntu 22.04")
+                 elseif(OS_VERSION STREQUAL "24.04")
+                     message(STATUS "This is Ubuntu 24.04")
+-                    set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/temp/compiler_libs/ubuntu24.04")
++                    set(PLUGIN_COMPILER_LIBS_DIR "${CMAKE_CURRENT_BINARY_DIR}/temp/compiler_libs/ubuntu24.04")
+                     set(PLUGIN_COMPILER_LIBS_URL "https://storage.openvinotoolkit.org/dependencies/thirdparty/linux/npu_compiler_vcl_ubuntu_24_04-7_5_0-a1ae54e9.tar.gz")
+                     set(PLUGIN_COMPILER_LIBS_TAR "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_ubuntu_24_04-${PLUGIN_COMPILER_VERSION}-a1ae54e9.tar.gz")
+                     set(PLUGIN_COMPILER_LIBS_DIR_EXTRACTED "${PLUGIN_COMPILER_LIBS_DIR}/npu_compiler_vcl_ubuntu_24_04-${PLUGIN_COMPILER_VERSION}-a1ae54e9")

--- a/ports/openvino/portfile.cmake
+++ b/ports/openvino/portfile.cmake
@@ -2,11 +2,13 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO openvinotoolkit/openvino
     REF "${VERSION}"
-    SHA512 97241f147c4e74054b787b1bb694ab1051df661ae049e75d0ffa54cec71325155e2a54ded777f09709f7e6a7ff740c7e69d2f3bd73320f1711330d07e427922b
+    SHA512 77259f2211aa27c70c4930795ab6a7f7a0eade6fa8cda9b38caafdb3fa8081dba7c6bb5c2fa75adbdb224e4d84c80c48f19eef463500cd58316aa55183e6d660
     HEAD_REF master
     PATCHES
+        windows_fix_giant_libs.patch
         onednn_gpu_includes.patch
         protobuf-6.patch
+        npu_deps.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
@@ -35,8 +37,8 @@ if(ENABLE_INTEL_GPU)
     vcpkg_from_github(
         OUT_SOURCE_PATH DEP_SOURCE_PATH
         REPO oneapi-src/oneDNN
-        REF 8edf6bb60fdaa2769f3cebf44bd2ee94da00744c
-        SHA512 4e413c93306d81b0c4823789990f9a99ce96cd7a523db9a347e399db0cdf0af3eb4aef449bc5821fe9023ae22655677b2be7897b9c4bfb974d11aab17b017d20
+        REF v3.11
+        SHA512 de60ecd881b97e9942441e0eb5c53e2caa2a0a1a1c78ab9211ab103244b66b62c0f3dfa5b322bb2c39dfe13f85a9aebf82b899dde1ccdc01ba8ff9deed832787
     )
     file(COPY "${DEP_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/plugins/intel_gpu/thirdparty/onednn_gpu")
 
@@ -49,8 +51,8 @@ if(ENABLE_INTEL_CPU)
     vcpkg_from_github(
         OUT_SOURCE_PATH DEP_SOURCE_PATH
         REPO openvinotoolkit/oneDNN
-        REF 3d7a6f1d068d8ae08f189aa4baa93d177bc07507
-        SHA512 f5de57f7a8972b4c3aefe359beeb51bd9a5cde6bbc6316891f73148555ef9f299145449faf7d761a8e801fb8ddee68b4455d2b9486067915fe7f445f6b099d6c
+        REF c6b79c1207bd5f20b9395536dab1d71a47cfcb1d
+        SHA512 2ed3444c60771229f051688964ee51b7cd229f75dfdbc6e59390d64223bb5d98074dd30cd4cd2458725bfba147bdf985bcc1d7ba8291f097ff4f291859b35ce3
     )
     file(COPY "${DEP_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/plugins/intel_cpu/thirdparty/onednn")
 
@@ -78,16 +80,16 @@ if(ENABLE_INTEL_CPU)
         vcpkg_from_github(
             OUT_SOURCE_PATH DEP_SOURCE_PATH
             REPO ARM-software/ComputeLibrary
-            REF v25.03
-            SHA512 a7c9f8138631aabe24cfe68021d3cdaf6716b69dbcf183694217ca87720efd399f5d809f9fd4522a435a6a991855bcf40d5c6fa6189d77ee8ca5caa1f9ade95c
+            REF v52.6.0
+            SHA512 b7f0c5b33466a064e87fbc549fbb78f49c03ef10d50d5e84a9e8dca66094e469cff5d6c5ec01e26873e957e74e8b8a8472c369ab8aeea74c7c36ad0dfe1ad152
         )
         file(COPY "${DEP_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/plugins/intel_cpu/thirdparty/ComputeLibrary")
 
         vcpkg_from_github(
             OUT_SOURCE_PATH DEP_SOURCE_PATH
             REPO ARM-software/kleidiai
-            REF eaf63a6ae9a903fb4fa8a4d004a974995011f444
-            SHA512 2eed2183927037ab3841daeae2a0df3dfaa680ae4dea5db98247d6d7dd3f897d5109929098eb1b08e3a0797ddc03013acdb449642435df12a11cffbe4f5d2674
+            REF v1.19.0
+            SHA512 46de1f0cdd04ce1e8de5d1bdb2499d07eb377e616eb3a8596fbcd296b7887e413be5470f383b5790cef73dc370bead3db36ef2ed116513b95924ae71d87ef123
         )
         file(COPY "${DEP_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/plugins/intel_cpu/thirdparty/kleidiai")
     endif()
@@ -101,8 +103,8 @@ if(ENABLE_INTEL_NPU)
     vcpkg_from_github(
         OUT_SOURCE_PATH DEP_SOURCE_PATH
         REPO intel/level-zero-npu-extensions
-        REF f8bba8915e0a5fe8317f7aa48007ecc5a8c179ca
-        SHA512 a93b907159c67fe76634869d71c5434756f32d6e6e81ae86cdc517499576dc5691221dde7821bbaf9d39bf3cd62066a032fd30be26e7662f2b78b046c0ddd2f6
+        REF 8404c63a88d182726038d2b07c219731dada9c21
+        SHA512 701c3dbb3fc016a5b2fd70b68375a8191013299b8824bdd2a2830efd69cadc3d4b104b6479c37a46164d1a1a9781e8cc50eebb96c0ada64743e5e2b9456bb1af
     )
     file(COPY "${DEP_SOURCE_PATH}/" DESTINATION "${SOURCE_PATH}/src/plugins/intel_npu/thirdparty/level-zero-ext")
 endif()
@@ -115,7 +117,7 @@ if(ENABLE_OV_TF_FRONTEND)
     list(APPEND FEATURE_OPTIONS "-DENABLE_SYSTEM_SNAPPY=ON")
 endif()
 
-if(ENABLE_OV_TF_LITE_FRONTEND)
+if(ENABLE_OV_TF_LITE_FRONTEND OR ENABLE_INTEL_NPU)
     list(APPEND FEATURE_OPTIONS "-DENABLE_SYSTEM_FLATBUFFERS=ON")
 endif()
 
@@ -132,7 +134,6 @@ vcpkg_cmake_configure(
         "-DCMAKE_DISABLE_FIND_PACKAGE_OpenCV=ON"
         "-DCPACK_GENERATOR=VCPKG"
         "-DENABLE_CLANG_FORMAT=OFF"
-        "-DENABLE_CPPLINT=OFF"
         "-DENABLE_JS=OFF"
         "-DENABLE_NCC_STYLE=OFF"
         "-DENABLE_PYTHON=OFF"
@@ -141,6 +142,7 @@ vcpkg_cmake_configure(
         "-DENABLE_SYSTEM_TBB=ON"
         "-DENABLE_TBBBIND_2_5=OFF"
         "-DENABLE_TEMPLATE=OFF"
+        "-DENABLE_PROFILING_ITT=OFF"
         "-DENABLE_OV_JAX_FRONTEND=OFF"
         "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}"
 )

--- a/ports/openvino/vcpkg.json
+++ b/ports/openvino/vcpkg.json
@@ -1,8 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "openvino",
-  "version": "2025.3.0",
-  "port-version": 1,
+  "version": "2026.0.0",
   "maintainers": "OpenVINO Developers <openvino@intel.com>",
   "summary": "This is a port for Open Visual Inference And Optimization toolkit for AI inference",
   "description": [
@@ -93,6 +92,15 @@
       "description": "Enables NPU plugin for inference",
       "supports": "x64 & !(osx | uwp)",
       "dependencies": [
+        {
+          "name": "flatbuffers",
+          "version>=": "2.0.6"
+        },
+        {
+          "name": "flatbuffers",
+          "host": true,
+          "version>=": "2.0.6"
+        },
         "level-zero"
       ]
     },
@@ -135,7 +143,7 @@
       "description": "Enables TensorFlow frontend for reading models in TensorFlow format",
       "dependencies": [
         {
-          "$comment": "to workaround a linking issue for 2025.2.0, 2025.3.0 openvino[core,tensorflow]",
+          "$comment": "to workaround a linking issue for 2026.0.0 openvino[core,tensorflow]",
           "name": "openvino",
           "default-features": false,
           "features": [
@@ -168,7 +176,7 @@
           "version>=": "2.0.6"
         },
         {
-          "$comment": "to workaround a linking issue for 2025.2.0, 2025.3.0 openvino[core,tensorflow]",
+          "$comment": "to workaround a linking issue for 2026.0.0 openvino[core,tensorflow]",
           "name": "openvino",
           "default-features": false,
           "features": [

--- a/ports/openvino/windows_fix_giant_libs.patch
+++ b/ports/openvino/windows_fix_giant_libs.patch
@@ -1,0 +1,59 @@
+diff --git a/src/plugins/intel_gpu/CMakeLists.txt b/src/plugins/intel_gpu/CMakeLists.txt
+index 427460fb59..818047190a 100644
+--- a/src/plugins/intel_gpu/CMakeLists.txt
++++ b/src/plugins/intel_gpu/CMakeLists.txt
+@@ -72,7 +72,7 @@ ov_add_plugin(NAME ${TARGET_NAME}
+ target_compile_options(${TARGET_NAME} PRIVATE
+     $<$<CONFIG:Release>:$<IF:$<CXX_COMPILER_ID:MSVC>,/Os,-Os>>)
+ 
+-target_link_libraries(${TARGET_NAME} PRIVATE openvino_intel_gpu_graph openvino::pugixml)
++target_link_libraries(${TARGET_NAME} PRIVATE openvino_intel_gpu_graph openvino_intel_gpu_graph_impls openvino::pugixml)
+ 
+ target_include_directories(${TARGET_NAME} PRIVATE
+         ${CMAKE_CURRENT_SOURCE_DIR}/include/)
+diff --git a/src/plugins/intel_gpu/src/graph/CMakeLists.txt b/src/plugins/intel_gpu/src/graph/CMakeLists.txt
+index 7538f01b3b..576c3fae29 100644
+--- a/src/plugins/intel_gpu/src/graph/CMakeLists.txt
++++ b/src/plugins/intel_gpu/src/graph/CMakeLists.txt
+@@ -116,8 +116,6 @@ ov_add_target(
+     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
+     EXCLUDED_SOURCE_PATHS
+         ${EXCLUDE_PATHS}
+-    OBJECT_FILES
+-        ${OBJ_FILES}
+     INCLUDES
+         PUBLIC
+             ${COMMON_INCLUDE_DIRS}
+@@ -139,3 +137,20 @@ ov_set_threading_interface_for(${TARGET_NAME})
+ set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
+ 
+ ov_install_static_lib(${TARGET_NAME} ${OV_CPACK_COMP_CORE})
++
++
++# Link all object files from ./impls/* into a different library to avoid giant files
++
++add_library(${TARGET_NAME}_impls STATIC ${OBJ_FILES})
++
++target_link_libraries(${TARGET_NAME}_impls PUBLIC ${COMMON_LINK_LIBRARIES} ${EXTRA_LINK_LIBRARIES} openvino::reference)
++
++add_dependencies(${TARGET_NAME}_impls run_codegen_test)
++
++target_compile_options(${TARGET_NAME}_impls PRIVATE ${COMMON_COMPILE_OPTIONS})
++
++ov_set_threading_interface_for(${TARGET_NAME}_impls)
++
++set_target_properties(${TARGET_NAME}_impls PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
++
++ov_install_static_lib(${TARGET_NAME}_impls ${OV_CPACK_COMP_CORE})
+diff --git a/src/plugins/intel_gpu/tests/unit/CMakeLists.txt b/src/plugins/intel_gpu/tests/unit/CMakeLists.txt
+index 09f95d298a..3b3685875b 100644
+--- a/src/plugins/intel_gpu/tests/unit/CMakeLists.txt
++++ b/src/plugins/intel_gpu/tests/unit/CMakeLists.txt
+@@ -69,6 +69,7 @@ endif()
+ set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
+ 
+ target_link_libraries(${TARGET_NAME} PRIVATE openvino_intel_gpu_graph
++                                             openvino_intel_gpu_graph_impls
+                                              OpenCL::OpenCL
+                                              gtest
+                                              gtest_main

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7321,8 +7321,8 @@
       "port-version": 0
     },
     "openvino": {
-      "baseline": "2025.3.0",
-      "port-version": 1
+      "baseline": "2026.0.0",
+      "port-version": 0
     },
     "openvpn3": {
       "baseline": "3.10",

--- a/versions/o-/openvino.json
+++ b/versions/o-/openvino.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6373049880cd0ffb4ec7ac796e9a1c59f2e2ccd0",
+      "version": "2026.0.0",
+      "port-version": 0
+    },
+    {
       "git-tree": "9385d70cc142992e9e10411d61555807f869d112",
       "version": "2025.3.0",
       "port-version": 1

